core/src/plugins/kubernetes/config.ts

```diff
@@ -465,6 +536,77 @@ export const kubernetesConfigBase = () =>
     clusterBuildkit: joi
       .object()
       .keys({
+        cache: joi
+          .array()
+          .items(buildkitCacheConfigurationSchema())
+          // TODO: fix default value, to include defaults automatically
+          .default([{ type: "registry", mode: "auto", tag: "_buildcache", export: true }])
+          .description(
+            dedent`
+            Use the \`cache\` configuration to customize the default cluster-buildkit cache behaviour.
+
+            The default value is:
+            \`\`\`
```

Comment:
Add `yaml` here (for syntax highlighting in the rendered docs).
```suggestion
            \`\`\`yaml
```